src/ai_hawk/job_manager.py

```diff
@@ -513,12 +500,12 @@ def job_tile_to_job(self, job_tile) -> Job:
             logger.warning(f"Failed to extract job ID: {e}", exc_info=True)
 
         try:
-            job.location = job_tile.find_element(By.CLASS_NAME, 'job-card-container__metadata-item').text
+            job.location = job_tile.find_element(By.XPATH, './/span[contains(normalize-space(), " · ")]').text.split(' · ')[-1].strip()
```

Comment:
this do works why mess with it?

Explain your xpath, how are you making sure that span is location? are there any strong identifier this will for sure will be location.